src/main/java/com/baidu/hugegraph/cmd/SubCommands.java

```diff
@@ -1043,8 +1043,8 @@ public GraphMode convert(String value) {
         public String convert(String value) {
             E.checkArgument(value != null && !value.isEmpty(),
                             "Strategy can't be null or empty");
-            E.checkArgument(AuthRestoreStrategy.STOP.string().equals(value) ||
-                            AuthRestoreStrategy.IGNORE.string().equals(value),
+            E.checkArgument(AuthRestoreConflictStrategy.STOP.string().equals(value) ||
```

Comment:
exceed 80 chars, we can add AuthRestoreConflictStrategy.STOP.match(value) method